modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA command line interface (CLI).
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+
+.Procedure
+
+* Create a machine pool with a custom tag by running the following command:
++
+--
+[source,terminal]
+----
+$ rosa create machinepools --cluster=<name> --replicas=<replica_count> \
+     --tags=<key>=<value>,<effect>,<key>=<value>,<effect> \ <1>
```

Comment:
The command is wrong.
Here is the help message and the usage.
```
lixue@Xue-Lis-MacBook-Pro ~ % rosa create machinepool -c 2bgu1fqljjne6sujk3v031n5gu5navur -h|grep tags
      --tags strings                            Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: 'key value, foo bar'
```
Below command can work
```
lixue@Xue-Lis-MacBook-Pro ~ % rosa create machinepool -c 2bgu1fqljjne6sujk3v031n5gu5navur --name tags --replicas 0 --tags 'xuelitagkey xuelitagv,xuelitagk2 xuelitagv2'
I: Checking available instance types for machine pool 'tags'
I: Machine pool 'tags' created successfully on cluster '2bgu1fqljjne6sujk3v031n5gu5navur'
I: To view the machine pool details, run 'rosa describe machinepool --cluster 2bgu1fqljjne6sujk3v031n5gu5navur --machinepool tags'
I: To view all machine pools, run 'rosa list machinepools --cluster 2bgu1fqljjne6sujk3v031n5gu5navur'
lixue@Xue-Lis-MacBook-Pro ~ % rosa describe machinepool --cluster 2bgu1fqljjne6sujk3v031n5gu5navur --machinepool tags

ID:                                    tags
Cluster ID:                            2bgu1fqljjne6sujk3v031n5gu5navur
Autoscaling:                           No
Replicas:                              0
Instance type:                         m5.xlarge
Labels:                                
Taints:                                
Availability zones:                    us-west-2a, us-west-2b, us-west-2c
Subnets:                               subnet-0c62a6073380056c2, subnet-054dfa1129e16e092, subnet-02c30fa5122a69ea0
Spot instances:                        No
Disk size:                             300 GiB
Additional Security Group IDs:         
Tags:                                  red-hat-clustertype=rosa, red-hat-managed=true, xuelitagk2=xuelitagv2, xuelitagkey=xuelitagv
```

Reply:
Updated the command here:
![image](https://github.com/openshift/openshift-docs/assets/16167833/d2c89b1c-8c95-4d9e-8efa-51216993ab70)